src/modules/fundingManager/depositVault/interfaces/IFM_DepositVault_v1.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
```

Comment:
Why is the pragma in some contracts ^0.8.0 but in others 0.8.23

Reply:
Actual contracts are all `0.8.23` (or should be), while interfaces are all `^0.8.0`, so other projects can import our interfaces easier without needing multiple compiler versions potentially. We just following the example of other projects here.